src/utils/geometry.rs

```diff
@@ -208,6 +208,7 @@ floating_point_coordinate_impl! {
  */
 
 /// A point as defined by its x and y coordinates
+#[repr(C)]
```

Comment:
Why do we need that?

Reply:
I currently don't, but that allows e.g. to read in a `Vec<Rectangle<i32, Logical>>` as a `&[i32]` in the OpenGL renderer.